tests/realtikvtest/addindextest/global_sort_test.go

```diff
@@ -108,3 +108,63 @@ func TestGlobalSortCleanupCloudFiles(t *testing.T) {
 	require.Equal(t, 0, len(statFiles))
 	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/disttask/framework/dispatcher/WaitCleanUpFinished"))
 }
+
+func TestGlobalSortMultiSchemaChange(t *testing.T) {
+	var err error
+	opt := fakestorage.Options{
+		Scheme:     "http",
+		Host:       gcsHost,
+		Port:       gcsPort,
+		PublicHost: gcsHost,
+	}
+	server, err := fakestorage.NewServerWithOptions(opt)
+	require.NoError(t, err)
+	server.CreateBucketWithOpts(fakestorage.CreateBucketOpts{Name: "sorted"})
+	sortStorageURI := fmt.Sprintf("gs://sorted/addindex?endpoint=%s&access-key=aaaaaa&secret-access-key=bbbbbb", gcsEndpoint)
+
+	store := realtikvtest.CreateMockStoreAndSetup(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("drop database if exists addindexlit;")
+	tk.MustExec("create database addindexlit;")
+	tk.MustExec("use addindexlit;")
+
+	tk.MustExec("create table t_rowid (a int, b bigint, c varchar(255));")
+	tk.MustExec("create table t_int_handle (a bigint primary key, b varchar(255));")
+	tk.MustExec("create table t_common_handle (a int, b bigint, c varchar(255), primary key (a, c) clustered);")
+	// tk.MustExec(`create table t_partition (a bigint primary key, b int, c char(10)) partition by hash(a) partitions 4;`)
```

Comment:
Why comment out partition table?

Reply:
Because it takes too long to complete... 4 partitions means 4x import times.